docs/source/aws-backend.rst

```diff
@@ -0,0 +1,94 @@
+
+:py:class:`~dcos_e2e.node.Node`\ s of :py:class:`~dcos_e2e.cluster.Cluster`\ s created by the :py:class:`~dcos_e2e.backends.AWS` backend distinguish between :py:attr:`~dcos_e2e.node.Node.public_ip_address` and :py:attr:`~dcos_e2e.node.Node.private_ip_address`.
+The :py:attr:`~dcos_e2e.node.Node.private_ip_address` refers to the internal network of the AWS stack which is also used by DC/OS internally.
+The :py:attr:`~dcos_e2e.node.Node.public_ip_address` allows for reaching AWS EC2 instances from the outside e.g. from the ``dcos-e2e`` testing environment.
+
+AWS Regions
+-----------
+
+When launching a cluster with Amazon Web Services there are a number of different regions to choose from where the cluster is launched.
+It is recommended to use ``us-east-1`` or ``us-east-2`` to keep the cost low.
+
+See the `AWS Regions and Availability Zones`_ for available regions.
+
+Restricting access to the cluster
+---------------------------------
+
+The AWS backend takes a parameter :paramref:`~dcos_e2e.backends.AWS.admin_location`.
+This parameter restricts the access to the AWS stack from the outside to a particular IP address range.
+The default value ``'0.0.0.0/0'`` will allow accessing the cluster from anywhere.
+It is recommended to restrict the address range to a subnet including the public IP of the machine executing tests with the AWS backend.
+For example ``<external-ip>/24``.
+
+Accessing cluster nodes
+-----------------------
+
+SSH can be used to access cluster nodes for the purpose of debugging if :paramref:`~dcos_e2e.backends.AWS.workspace_dir` is set.
+The AWS backend generates a SSH key file ``id_rsa`` in a cluster-specific sub-directory under the :paramref:`~dcos_e2e.backends.AWS.workspace_dir` directory. The sub-directory is named after the unique cluster ID generated during cluster creation. The cluster ID is prefixed with ``dcos-e2e-`` and can be found through the DC/OS UI in the upper left corner or through the CCM UI when using `maws`_ with a Mesosphere AWS account.
+Adding this key to the ``ssh-agent`` or manually providing it via the ``-i`` flag after changing its file permissions to ``400`` will allow for connecting to the cluster via the ``ssh`` command.
+The SSH user depends on the :paramref:`~dcos_e2e.backends.AWS.linux_distribution` given to the :py:class:`~dcos_e2e.backends.AWS` backend.
+For :py:obj:`~dcos_e2e.distributions.Distribution.CENTOS_7` that is ``centos``.
+
+It is important to keep in mind files in the given :paramref:`~dcos_e2e.backends.AWS.workspace_dir` are temporary and are removed when the cluster is destroyed.
+If :paramref:`~dcos_e2e.backends.AWS.workspace_dir` is unset the :py:class:`~dcos_e2e.backends.AWS` backend will create a new temporary directory in an operating system specific location.
+
+Cluster lifetime
+----------------
+
+The cluster lifetime is fixed at two hours.
+
+If the cluster was launched with `maws`_ (Mesosphere temporary AWS credentials) the cluster can be controlled via `CCM`_.
```

Comment:
❓ Why define `maws` here and not above? Personally I'd just not bother with the definition now that it is a link.

Reply:
"not above" = not when you first mention `maws` in this document.